src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectsParser.java

```diff
@@ -44,4 +45,8 @@ public ProjectsParser(Gson gson) {
     public ProjectInfo parseSingleProjectInfo(JsonElement result) {
         return gson.fromJson(result, ProjectInfo.class);
     }
+
+    public String generateProjectInput(ProjectInput input) {
```

Comment:
I think it's better to have it in the parser. It provides a separation of concerns, and decouples the usage of gson. I also don't want to inject gson into the Client class, nor does it seem reasonable to make it available through a getter from the parser. Compared with those two alternatives, this is a better way. I am open to suggestions, though.